Event/Away Site Sprite Spring Cleaning

alsoandanswer opened the pull request:
1. Exorcises fake decal props. Generally horrible, and tends to break after a while (Most of the mapping changes are basically just fixing misaligned sprites and search + replacing fake decal props)
2. Touches up some old event sprites so they can be reused in future.
3. Genericises some Konyang props and assets so they may be used in future events more flexibly.

Sprites that are not mine from Mojave Sun, with some minor modifications.

There's a lot more I would like to finish, but this is all I could muster before the end of the month.
I have further plans to expand upon this for the future Unathi/Skrell/Whatever arcs. Makes my life easier.

alsoandanswer commented:
!review

DreamySkrell commented:
This should be separated into multiple PRs cause it's just touching a ton of things at once, including mapping, on which there are open PRs already - this is going to create merge conflicts for them (or for this PR depending on what is merged first).

DreamySkrell commented:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/5ea157ca-4aa0-4d48-b2df-f43ae5d129e9)

Looking at mapdiffbot2 renders, the new sprites are kinda. Do not look correct placement-wise. Like the lamp posts look fine as far as the sprite goes I guess, but they're in the middle of the tile now, so they don't allow for 1-tile-wide sidewalks. You would not expect to walk there.

![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/a0d3dabb-103b-4a6b-99ab-7b43f7bfb303)

Or the chainlink fences look like fine sprites, but the E/W sprite is in the middle of the tile, so it does not connect to N/S fences. So it looks really awkward.

![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/1eb0a88c-5252-4a59-b744-4baa42a88b4f)

Or this here.

alsoandanswer commented:
> ![image](https://private-user-images.githubusercontent.com/107256943/300556542-5ea157ca-4aa0-4d48-b2df-f43ae5d129e9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MDk4MzYsIm5iZiI6MTcwNjYwOTUzNiwicGF0aCI6Ii8xMDcyNTY5NDMvMzAwNTU2NTQyLTVlYTE1N2NhLTRhYTAtNGQ0OC1iMmRmLWY0M2FlNWQxMjllOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMFQxMDEyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05ZGIyZDE4YzY4ZTIxZGU4NjJjYzFkY2Y1MGNlYTFjOGRlZGJkMGIxOTYxYTRiNjk3MzA3OWM0ZmY0ODZkNDhlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.O-BPhQCQOfREWi4h8vYJOvAFlPGA7LZwMGjXz7_McKk)
> 
> Looking at mapdiffbot2 renders, the new sprites are kinda. Do not look correct placement-wise. Like the lamp posts look fine as far as the sprite goes I guess, but they're in the middle of the tile now, so they don't allow for 1-tile-wide sidewalks. You would not expect to walk there.
> 

I feel that is more of a problem of the map than the sprite itself. Sprites should not occupy intermediaries : it's harder to tell which tile it's on. Also, it adds the issue of pixelshifting everything manually, which is bad.

> ![image](https://private-user-images.githubusercontent.com/107256943/300556793-a0d3dabb-103b-4a6b-99ab-7b43f7bfb303.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MDk4MzYsIm5iZiI6MTcwNjYwOTUzNiwicGF0aCI6Ii8xMDcyNTY5NDMvMzAwNTU2NzkzLWEwZDNkYWJiLTEwM2ItNGE2Yi05OWFiLTdiNDNmN2JmYjMwMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMFQxMDEyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zYmNlMzU1NWVkY2VmMzQ5ZGM4YWUyMzMzMjE4YTk4YjgyYjg4MGY1ODg4NWE3MDc4YmMwNDlhYTE2ZjRiYzBkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.IKI1kzfU7gP4dDhKSS5jF1Bzn-uRYSKswvaKBHOvK3w)
> 
> Or the chainlink fences look like fine sprites, but the E/W sprite is in the middle of the tile, so it does not connect to N/S fences. So it looks really awkward.
> 

I've also added corner and junction connection sprites to help with this, however, this requires some maps to adjust their fences, since I've changed how they hitbox.

> ![image](https://private-user-images.githubusercontent.com/107256943/300558015-1eb0a88c-5252-4a59-b744-4baa42a88b4f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY2MDk4MzYsIm5iZiI6MTcwNjYwOTUzNiwicGF0aCI6Ii8xMDcyNTY5NDMvMzAwNTU4MDE1LTFlYjBhODhjLTUyNTItNGE1OS1iNzQ0LTRiYWE0MmE4OGI0Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMFQxMDEyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1mYTk3MThmNzVhODJjYjNlZTAzMWE5NDkwZjNiYWExZDQwMjY2NzYxZjI2ODcyZTc3Mzg0M2Q4ZGFhYzQ4YWVjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.jgeE75D3XDaAChhAusgMFwqXFPVKhl8quO_FLKE1tU4)
> 
> Or this here.

Again, can be fixed by moving the object.

TheDanseMacabre commented:
This is such a tremendous visual downgrade that I'm left baffled by this PR. I originally glanced over it and didn't look too deeply into it - the actual meaningful changes to this PR are hardly mentioned in the OP, so I didn't notice them at first. These sprites are from a Fallout server and look extremely ill-fitting, and the idea that it's a good move to delete quality original sprites that haven't even been given a chance to appear ingame to replace them with ill-fitting ports is, and always has been, a very bad idea. I very much hope this PR is closed.

DreamySkrell commented:
> I feel that is more of a problem of the map than the sprite itself. Sprites should not occupy intermediaries : it's harder to tell which tile it's on. Also, it adds the issue of pixelshifting everything manually, which is bad.

> I've also added corner and junction connection sprites to help with this, however, this requires some maps to adjust their fences, since I've changed how they hitbox.

> Again, can be fixed by moving the object.

That should all be fixed in this PR.

A PR can't just change the functionality of things (like changing hitboxes, or the size/position of the sprite on a turf) without fixing the maps they're used on. Especially since the event arc has already started, it's passed the deadline now, and all the assets should be finished by now.

As for pixelshifting, I think it's fine for decorative things like lamp posts.

DreamySkrell commented:
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/12d28fe6-1035-4991-8610-a009fce90616)
![image](https://github.com/Aurorastation/Aurora.3/assets/107256943/6f97f187-015f-403c-9d03-3e8a4b52b826)

Absolutely against adding real-life american road signs or game consoles. They don't fit on Konyang, or anywhere on the server.

alsoandanswer commented:
> ![image](https://private-user-images.githubusercontent.com/107256943/301195290-12d28fe6-1035-4991-8610-a009fce90616.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3MTQzOTgsIm5iZiI6MTcwNjcxNDA5OCwicGF0aCI6Ii8xMDcyNTY5NDMvMzAxMTk1MjkwLTEyZDI4ZmU2LTEwMzUtNDk5MS04NjEwLWEwMDlmY2U5MDYxNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMVQxNTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zNGUwMjlhNzYwM2MyODUzNzU3Y2ZmNDE2YjAyYjA4MGJkNWNmMTkxZjQ1OTBmZTI1MGMxOGZjNDFkNGU2MTdmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.RsP4h5UQTIfUokUvzxtMxwjo4gRzUDPj4wxBPn-PgjM) ![image](https://private-user-images.githubusercontent.com/107256943/301195304-6f97f187-015f-403c-9d03-3e8a4b52b826.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY3MTQzOTgsIm5iZiI6MTcwNjcxNDA5OCwicGF0aCI6Ii8xMDcyNTY5NDMvMzAxMTk1MzA0LTZmOTdmMTg3LTAxNWYtNDAzYy05ZDAzLTNlOGE0YjUyYjgyNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEzMVQxNTE0NThaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lMWM1Yjk0NzM5YzcxMGFjNGM0OGI5Njk1ZjE5YjEwN2VhMDY2NjNlOGJmYWQzMmNkZjhhNDEwODM0NTYyYWU1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.K_Bck0TDTgJoeaSCjyCTsAt4zswg1S9rB63Ug_SnFpg)
> 
> Absolutely against adding real-life american road signs or game consoles (XBOX and PlayStation). They don't fit on Konyang, or anywhere on the server.

The highway signs aren't used anywhere on Konyang. They're just there as an additional asset, whosoever wants to use it for an appropriate project. Just because it isn't appropriate *now* as grounds for rejection is counterproductive to building an extensive asset library for future projects.

The game consoles are loose approximations, and they're extremely uncommon props used only a handful of times on the entire server. I really don't think they pose that much of a threat to the suspension of disbelief.

FluffyGhoster commented on code/game/objects/structures/decorative.dm:
```diff
@@ -0,0 +1,96 @@
+	desc = "A bland air conditioning unit."
+	icon = 'icons/obj/structure/miscellaneous.dmi'
+	icon_state = "ac"
+	layer = 2.7
```
```suggestion
	layer = BELOW_OBJ_LAYER
```
Probably

FluffyGhoster commented on code/game/objects/structures/decorative.dm:
```diff
@@ -0,0 +1,96 @@
+	icon = 'icons/obj/structure/miscellaneous.dmi'
+	icon_state = "ac"
+	layer = 2.7
+	density = 1
```
```suggestion
	density = TRUE
```

FluffyGhoster commented on code/game/objects/structures/decorative.dm:
```diff
@@ -0,0 +1,96 @@
+	desc = "The *other* kind of power-point."
+	icon = 'icons/obj/structure/miscellaneous.dmi'
+	icon_state = "outlet"
+	layer = 2.7
```
```suggestion
	layer = LAYER_UNDER_TABLE
```

FluffyGhoster commented on code/game/objects/structures/decorative.dm:
```diff
@@ -0,0 +1,96 @@
+	desc = "A nondescript vent."
+	icon = 'icons/obj/structure/miscellaneous.dmi'
+	icon_state = "vent"
+	layer = 2.7
```
```suggestion
	layer = LAYER_UNDER_TABLE
```

FluffyGhoster commented on code/game/objects/structures/decorative.dm:
```diff
@@ -0,0 +1,96 @@
+	icon = 'icons/obj/structure/64x64_misc.dmi'
+	icon_state = "umbrella"
+	anchored = TRUE
+	layer = 9
```
Pick a define or add one for this

FluffyGhoster commented on code/game/objects/structures/decorative.dm:
```diff
@@ -0,0 +1,96 @@
+	icon_state = "crane"
+	anchored = TRUE
+	density = TRUE
+	layer = 9
```
As above

FluffyGhoster commented on code/game/objects/structures/light_pole.dm:
```diff
@@ -16,65 +16,83 @@
 
+/obj/effect/overlay/street_light/classic
+	icon_state = "classic_lamp_light"
+	density = 1
```
```suggestion
	density = TRUE
```

FluffyGhoster reviewed (approved):
Code LGTM
For what I'm able to tell, the maps also look good, but I'm not an expert about them

DreamySkrell commented:
> The interstate signs aren't used anywhere on Konyang. They're just there as an additional asset, whosoever wants to use it for an appropriate project. Same train of thought with genericising things. Just because it isn't appropriate _now_ as grounds for rejection is counterproductive to building an extensive asset library for future projects.
> 
> The game consoles are loose approximations, and they're extremely uncommon props used only a handful of times on the entire server. I really don't think they pose that much of a threat to the suspension of disbelief.

Assets are nice, yes. But I don't think we should be adding stuff just because it may be used some time in the future, maybe years from now. I do not see a place in the lore that could be using modern day american road signs anywhere. Not in Biesel, not even in Sol, except if we literally visited the USA (if it still exists). 

Otherwise, why not just add every asset from every other server? Someone may want to use them 5 or 10 years from now, who knows.

As for the XBOX and Playstation, just, no.

I cannot approve this PR as it is. I do not think American road signs or game consoles are a good addition to the repo.

Arrow768 commented:
@alsoandanswer You have mentioned this in the PR description:

Genericises some Konyang props and assets so they may be used in future events more flexibly.

Could you explain why that is necessary and give a few examples where existing sprites can not be used for other locations.

alsoandanswer commented:
> @alsoandanswer You have mentioned this in the PR description:
> 
> Genericises some Konyang props and assets so they may be used in future events more flexibly.
> 
> Could you explain why that is necessary and give a few examples where existing sprites can not be used for other locations.

1. Road signs and Traffic Lights/Street Lamps
Namely the stop sign had Korean text on it, which meant it couldn't be used outside of Konyang.
They were also *very* oversaturated, so I had to replace them.
The more muted colors mean they fit into a lot more settings without them glaringly standing out.

2. Signs and whatnot
They use Konyang text, which is fine here. I just adjusted them to use a standardized 64x32 format.
Some props were electronic or holograms, which predisposed them unable to be used in maps/areas without electricty.

3. Hotel Stuff and other fake decal props
They used fake decals which have a tendency to break and be missed in unit tests.
I made them proper objects and did some QC such as standardizing palette and touch-ups.

The rest was mostly just QC.

Leroyengloys commented:
Synth lore here, it is to my understanding that the PR replaces sprites that were specifically made for the arc, without any consultation from synthlore. Apart from the fact that the ported material is a poor choice in my opinion, I would request that if sprites must absolutely be replaced, that it happens after we are done with Silicon Nightmares.

alsoandanswer commented:
> Synth lore here, it is to my understanding that the PR replaces sprites that were specifically made for the arc, without any consultation from synthlore. Apart from the fact that the ported material is a poor choice in my opinion, I would request that if sprites must absolutely be replaced, that it happens after we are done with Silicon Nightmares.

It's a wrong understanding. These are just QC changes. They are fundementally unchanged.
Please don't stonewall this any further. The merge conflicts will keep piling up.

alsoandanswer commented:
Removed the interstate sign and games console references

Leroyengloys commented:
If you wish to add more signs for more general props, that is fine. But don't replace the existing ones used in Konyang. The blue colour on the arrow sign, the script on the stop sign, are all conscious choices that were made for the konyang aesthetic and I really don't think the quality gets better with the replacements. 

Also we were never informed, so I don't believe voicing the opinion of those most directly affected by it is stonewalling.

alsoandanswer commented:
> If you wish to add more signs for more general props, that is fine. But don't replace the existing ones used in Konyang. The blue colour on the arrow sign, the script on the stop sign, are all conscious choices that were made for the konyang aesthetic and I really don't think the quality gets better with the replacements.

They have been touched up and readded.

DreamySkrell reviewed (approved):
code and mapping looks fine
